site/helpers/steamid.php

```diff
@@ -101,6 +101,22 @@ public static function getType()
         return (!$user->get('guest')) ? 'logout' : 'login';
     }
 
+    /**
+     * Return 1 if https url, 2 if http,
+     * -1 if neither.
+     * Used when creating return URL with JRoute().
+     */
+    protected function usessl_int() {
```

Comment:
It would be very nice if we can use camelCase for function name here to match with the general code style.